package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "fecha",
-  "version": "2.2.0",
+  "version": "2.2.1",
```

Comment:
Yes, sorry, this change should have probably been in a separate commit in the first place. Will return it back to 2.2.0